### PR TITLE
Send message to users who owe distribution funds

### PR DESCRIPTION
**Pull request by chair28980**
Send message to users who owe distribution funds

### CODE REVIEW

**Comment by chair28980** [comment]
@youngkidwarrior I need a lil boost with construction the discord.js client message.

**Comment by youngkidwarrior** [comment]
@chair28980 You'll have to set some criteria to run the script. So from within `index.js` you can set it to run when a command is received or set it to run when the bot starts up.

File: src/distributionPaybackMessage.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    let contents = await fs.readFile('./owed.csv', {encoding: 'utf8'})
+    contents = contents.trim().split("\n")
+
+    for (content of contents) {
+        let [address, balanceOwed] = content.split(",")
+        let user = addressBook.find(function findUser(user) {
+            return user.address === address
+        })
+
+        console.log(user.discordId)
```
Comment (youngkidwarrior) [inline src/distributionPaybackMessage.js]: Discord.js has a `client.guild.members.cache.get(discordId)` function that returns Discords User object. You can then send a message with user.send()

File: src/distributionPaybackMessage.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const { promises: fs } = require('fs')
+const Discord = require("discord.js")
+const client = new Discord.Client()
```
Comment (youngkidwarrior) [inline src/distributionPaybackMessage.js]: starting a new client here wont work. Call this function from index.js and pass it the client through function params

File: src/distributionPaybackMessage.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+const addressBook = require("./addressbook.json")
+const daoAddress = 'theDAOaddress'
```
Comment (youngkidwarrior) [inline src/distributionPaybackMessage.js]: add the DAO address

**Review by youngkidwarrior — CHANGES_REQUESTED** [review]

**Pull request merged**